src/uisMultipleDirective.js

```diff
@@ -0,0 +1,393 @@
+uis.directive('uisMultiple', ['uiSelectMinErr','$timeout', function(uiSelectMinErr, $timeout) {
```

Comment:
Is this really the directive name you want?

    <uis-multiple>

Aren't you afraid of the confusion it will cause by being in a totally different namespace from the other directives in this same library?